main/src/main/java/com/sedmelluq/discord/lavaplayer/source/nico/NicoAudioSourceManager.java

```diff
@@ -80,8 +84,6 @@ public AudioItem loadItem(AudioPlayerManager manager, AudioReference reference)
     }
 
     private AudioTrack loadTrack(String videoId) {
-        checkLoggedIn();
```

Comment:
do logins timeout? or do they have to be refreshed?

Reply:
I would assume they're cookie based thus expire when the cookie does. However, accounts aren't necessary to use NicoNico anymore.